### PR TITLE
Fix/revoke short lived tokens

### PR DESCRIPTION
**Pull request by fisher60**
Token expiration time does not appear to be working from env vars, this needs to be tested prior to merging. Seems env vars from .env in deployment are not taking effect in this project, nor is .env at project level.

### CODE REVIEW

File: abandonauth/dependencies/auth/jwt.py [hunk 1]
```diff
@@ -19,6 +22,9 @@ def generate_jwt(user_id: str) -> str:
         key=settings.JWT_SECRET.get_secret_value(),
         algorithm=settings.JWT_HASHING_ALGO
     )
+
+    valid_token_cache[token] = None
```
Comment (GDWR) [inline abandonauth/dependencies/auth/jwt.py:26]: Why are we using a none value here? Can we just use a `set`?

Comment (fisher60) [reply abandonauth/dependencies/auth/jwt.py:26]: I understand that a dictionary should be the most performant for this application, `None` is the smallest type I know of in python so it is being used as a placeholder.

If a set would be more performant, than I am open to changing it, although I do not think that would be the case.

Comment (GDWR) [reply abandonauth/dependencies/auth/jwt.py:26]: It's more about using the right tool for the job, we don't need values. So let's use a set. I would be surprised if a dictionary was more performant in this usage as they should both be hashmaps, but this won't be very important and we are already using Python :laughing:

Comment (fisher60) [reply abandonauth/dependencies/auth/jwt.py]: Updated to use set

**Pull request merged**